### ISSUE TITLE
Filter based on vote criteria: good, bad unsure

### ISSUE THREAD

**Issue by mondana** [chime-experiment/bondia#123]
It would be useful to see all the "good" days first.

**Comment by wulfda02** [comment]
I don't think it matters whether good days are first (or bad, or unsure), but it would be nice to sort according to vote first, and day second (as sorting according to number of votes currently works).  That makes it pretty easy to find the boundaries between the different categories in the drop down menu: e.g. [1,4,5,9,3,7,8,2,6] = [(1,4,5,9),(3,7,8),(2,6)]. There are at least two use cases I can think of:

1.  For training new users, this would facilitate looking though bad days to see the different kinds of pathologies. I effectively did this by brute force when I took screen shots from a bunch of bad days and put them into a powerpoint.
2. For discussing 'unsure' days.  After several users have had a first pass at voting, they could flip through all the 'unsure' days and discuss as a group what the final classification should be (e.g. during a pipeline call)

**Comment by wulfda02** [comment]
Where there are multiple votes in conflict for a particular day, we'll need to decide on a logic.  For use case 2 above, I think that any day with at least one good or a bad vote should not be sorted as unsure.  Otherwise, we'll need to change everyone's vote before it is removed from the unsure list. Additionally, the most conservative thing to do is to sort a day as bad if it has at least on bad vote.  Something like this:
```
if 'bad' in votes:
    day = 'bad'
elif 'good' in votes:
    day = 'good'
elif 'unsure' in votes:
    day = 'unsure'
else:
    day = 'no vote'
```
